iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/config/constant/PipeConnectorConstant.java

```diff
@@ -242,6 +242,10 @@ public class PipeConnectorConstant {
                   CONNECTOR_LOAD_TSFILE_STRATEGY_ASYNC_VALUE,
                   CONNECTOR_LOAD_TSFILE_STRATEGY_SYNC_VALUE)));
 
+  public static final String PIPE_ALREADY_EXIST_MSG =
+      "the pipe with the same name has been created";
+  public static final String PIPE_NOT_EXIST_MGS = "the pipe does not exist";
+
```

Comment:
These constant may not belong to PipeConnector. I think it's better to place them in `ConsensusPipeDataNodeDispatcher`

Reply:
`ConsensusPipeDataNodeDispatcher` belong to DataNode, and cannot be used in ConfigNode

Reply:
ok, I will try to figure it out